README.md

```diff
@@ -48,34 +47,33 @@ optional arguments:
 `ddit` is used to build two major categories of DAR files:
 applications and integrations. Applications are simple composites of
 one or more distinct sub-artifacts. This is useful to deploy, for
-example, a DAML model alongside the Python Bots and UI code composing
+example, a Daml model alongside the Python Bots and UI code composing
 the rest of the application. When building these sorts of DIT files,
 `ddit` primarily serves to assemble packages out of components built
 by other build processes. Put another way, `ddit` won't build your
 user interface itself, it has to be built before `ddit` can package
 it into a DIT file.
 
 For examples of what this looks like in practice, please see one of
-several sample applications available through DABL:
+several sample applications available through Daml Hub:
 
-* <https://github.com/digital-asset/dablchat>
-* <https://github.com/digital-asset/dablchess>
-* <https://github.com/OpenSaaSame/board>
+- <https://github.com/digital-asset/dablchat>
+- <https://github.com/digital-asset/dablchess>
+- <https://github.com/OpenSaaSame/board>
```

Comment:
Does this have any semantic importance?

Reply:
Ah, Prettier has some formatting rules for markdown files, and I have a VS Code extension that auto-runs Prettier on save (which I guess runs regardless of if the open project is actually using Prettier)